testing/trino-product-tests/src/main/java/io/trino/tests/iceberg/TestSparkCompatibility.java

```diff
@@ -161,12 +168,12 @@ public void testPrestoCreatesSparkDrops()
         onSpark().executeQuery("DROP TABLE " + sparkTableName(baseTableName));
     }
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
-    public void testSparkReadsPrestoPartitionedTable()
+    @Test(dataProvider = "storage_formats", groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testSparkReadsPrestoPartitionedTable(String format)
     {
-        String baseTableName = "test_spark_reads_presto_partitioned_table";
+        String baseTableName = "test_spark_reads_presto_partitioned_table_" + format;
         String prestoTableName = prestoTableName(baseTableName);
-        onPresto().executeQuery(format("CREATE TABLE %s (_string VARCHAR, _bigint BIGINT) WITH (partitioning = ARRAY['_string'])", prestoTableName));
+        onPresto().executeQuery(format("CREATE TABLE %s (_string VARCHAR, _bigint BIGINT) WITH (partitioning = ARRAY['_string'], format = '" + format + "')", prestoTableName));
```

Comment:
```suggestion
        onPresto().executeQuery(format("CREATE TABLE %s (_string VARCHAR, _bigint BIGINT) WITH (partitioning = ARRAY['_string'], format = '%s')", prestoTableName, format));
```
sorry if it was confusing, meant a change like the above in https://github.com/trinodb/trino/pull/6699#discussion_r564857535